omnibus/cookbooks/omnibus-supermarket/templates/default/rails.nginx.conf.erb

```diff
@@ -85,6 +85,10 @@ server {
     break;
   }
 
+  location ~ ^/curry/ {
+    return 410;
```

Comment:
Fast-fail krufty curry webhooks without making the Rails app think about it. While this kinda obviates the need for the Rails app to also handle this route, I've left an entry in routes.rb with the same behavior Just In Case™.